foodalert/test/test_subscription.py

```diff
@@ -339,6 +372,34 @@ def test_patch_subscription(self, email=None, sms=None):
         data = response.json()
         self.assertNotEqual(payload2['email'], data['email'])
 
+    @parameterized.expand(VALID_TEST_CASES)
+    @transaction.atomic
+    def test_wrong_user_patch_subscription(self, email=None, sms=None):
+        """
+        A patch request to update either sms or email is valid. The fields in
```

Comment:
modify method description

Reply:
updated it to better describe the function